src/components/Banners/SiteBanner.tsx

```diff
@@ -25,18 +23,22 @@ export const SiteBanner = () => {
     Cookies.set(text, "viewed", { expires: 7 });
   };
 
-  return (
-    showBanner && (
-      <DismissibleBanner bannerTheme={theme} data-cy="sitewide-banner">
-        <span>{jiraLinkify(text, jiraHost)}</span>
-        <IconButton
-          aria-label="Close Site Banner"
-          onClick={hideBanner}
-          data-cy="dismiss-sitewide-banner-button"
-        >
-          <Icon glyph="X" />{" "}
-        </IconButton>
-      </DismissibleBanner>
-    )
-  );
+  const variant = mapThemeToVariant?.[theme] ?? Variant.Info;
```

Comment:
bump, since it passes the tests written
```suggestion
  const variant = mapThemeToVariant[theme] ?? Variant.Info;
```